db/migrate/20221206192349_remove_appropriate_body_unknown_from_school_cohorts.rb

```diff
@@ -0,0 +1,9 @@
+# frozen_string_literal: true
+
+class RemoveAppropriateBodyUnknownFromSchoolCohorts < ActiveRecord::Migration[6.1]
+  def change
+    safety_assured do
+      remove_column :school_cohorts, :appropriate_body_unknown, :boolean
```

Comment:
in the cohort setup there's a step in which the user can set the AB as unknown.
I'm not sure where we're saving the choice now.